notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -759,7 +730,7 @@
         "id": "910541af051c"
       },
       "source": [
-        "The following generates a JSON file that you'll use to run the pipeline:"
+        "The following generates a JSON file that is used to run the pipeline:"
```

Comment:
- change: "The following generates a JSON file that is used to run the pipeline:"
    - to: "The following generates a JSON file that is then used to run the pipeline:"